packages/shared/src/cards.ts

```diff
@@ -1,13 +1,28 @@
+import { words } from 'lodash';
+
 export const Suit = {
   Golds: 'Golds',
   Cups: 'Cups',
   Clubs: 'Clubs',
   Swords: 'Swords',
 } as const;
 
-export type Suit = typeof Suit[keyof typeof Suit];
+export const VALUES: { [key: number]: string } = {
+  1: 'Ace',
+  2: 'Two',
+  3: 'Three',
+  4: 'Four',
+  5: 'Five',
+  6: 'Six',
+  7: 'Seven',
+  8: 'Knave',
+  9: 'Knight',
+  10: 'King',
+};
+
+export type SuitType = typeof Suit[keyof typeof Suit];
 
-export type Card = { value: number; suit: Suit };
+export type Card = { value: number; suit: SuitType };
```

Comment:
there are few other places where I'm doing this, exporting with the same name, maybe we should rename them all if you are against this bad practice

Reply:
Don't you think it is? How come TS doesn't get confused...

Reply:
types are compile time only, ts is probably able to use one or the other, I think the right term in ts is overload but not sure because that applies to function usually. In client we have an eslint rule that prevent this to happen, but in shared it just worked so I thought it was smart to keep it like that